src/main/kotlin/io/github/nomisrev/routes/error.kt

```diff
@@ -1,17 +1,7 @@
 package io.github.nomisrev.routes
 
 import arrow.core.Either
-import io.github.nomisrev.CannotGenerateSlug
-import io.github.nomisrev.DomainError
-import io.github.nomisrev.EmailAlreadyExists
-import io.github.nomisrev.EmptyUpdate
-import io.github.nomisrev.IncorrectInput
-import io.github.nomisrev.IncorrectJson
-import io.github.nomisrev.JwtGeneration
-import io.github.nomisrev.JwtInvalid
-import io.github.nomisrev.PasswordNotMatched
-import io.github.nomisrev.UserNotFound
-import io.github.nomisrev.UsernameAlreadyExists
+import io.github.nomisrev.*
```

Comment:
I don't used star imports in any of my project.

```suggestion
port io.github.nomisrev.CannotGenerateSlug
import io.github.nomisrev.DomainError
import io.github.nomisrev.EmailAlreadyExists
import io.github.nomisrev.EmptyUpdate
import io.github.nomisrev.IncorrectInput
import io.github.nomisrev.IncorrectJson
import io.github.nomisrev.JwtGeneration
import io.github.nomisrev.JwtInvalid
import io.github.nomisrev.PasswordNotMatched
import io.github.nomisrev.UserNotFound
import io.github.nomisrev.UsernameAlreadyExists
```